### DISCUSSION TITLE
How to access the result of send?

### DISCUSSION THREAD

**Discussion by mkonzal** [smallrye/smallrye-reactive-messaging#2082] [Q&A]
When I'm using pure KafkaClient then the `producer.send()` returns `Future<RecordMetadata>`. In MP reactive messaging the `emitter.send()` returns `CompletableFuture<Void>`.
Is there any way in MP reactive messaging to access the result of send?

Thanks.

**Comment by cescoffier** [comment]
No, the emitter will only tell you when the write has been acknowledged by the broker.

If you need the metadata, you can use the underlying kafka client.
